packages/fxa-auth-server/lib/routes/utils/signup.js

```diff
@@ -65,6 +65,7 @@ module.exports = (log, db, mailer, push, verificationReminders, glean) => {
 
       // Our post-verification email is very specific to sync,
       // so only send it if we're sure this is for sync or sync scoped client.
+      // Do not send for browser signiins that are not sync
```

Comment:
~signiins~ signins

I think this email might still hit 'sync' conditions for `service=relay` since it requests the Sync scope? I think we can verify this if you check out my branch in #17869, cherry-pick this on top of it, and go through the relay signup flow as instructed there. (Sorry also if your changes in `packages/fxa-auth-server/lib/senders/email.js` account for this, I just haven't checked into your branch yet 🙈)

Reply:
I see what you mean and updated to account for services other than sync. While testing, I also noticed that the first condition was always resolving as false because we are passing the client id from settings and not the service (e.g., 'sync' or 'relay') - I am working on updating was is passed from fxa-settings.